app/src/main/java/de/tum/informatics/www1/artemis/native_app/android/db/AppDatabase.kt

```diff
@@ -29,7 +29,7 @@ import de.tum.informatics.www1.artemis.native_app.feature.push.communication_not
         CommunicationMessageEntity::class
     ],
     exportSchema = true,
-    version = 10,
+    version = 11,
```

Comment:
With the latest release the database version was increased to 11, to update it to 12 here :)